pkg/services/govmomi/util.go

```diff
@@ -58,46 +58,80 @@ func sanitizeIPAddrs(ctx *context.VMContext, ipAddrs []string) []string {
 //   3. If it is not found by instance UUID, fallback to an inventory path search
 //      using the vm folder path and the VSphereVM name
 func findVM(ctx *context.VMContext) (types.ManagedObjectReference, error) {
+	var err error
+
+	moRef, biosUUIDMatch, err := findVMWithBiosUUID(ctx)
+	if biosUUIDMatch {
+		return moRef, err
+	}
+
+	moRef, err = findVMWithInstanceUUID(ctx)
+	if err == nil {
+		return moRef, nil
```

Comment:
replace nil by `err`

Reply:
This was suggested by the linter, since at this point err is known to be nil.